README/images.md

```diff
@@ -0,0 +1,34 @@
+### Uploading Images
+Uploading an image through the Joplin CMS in staging or production will upload that image to the CMS_MEDIA s3 images bucket. And it also add entries for that image to the `base_translatedimage` and `base_translatedimagerendition` database tables. `base_translatedimage` is the primary image information table, and `base_translatedimagerendition` is a child table that contains all the different image sizes that are available on s3.
+
+For example:
+```
+base_translatedimage:
+id |      title       |               file
+----+------------------+----------------------------------
+21 | love-chicken.jpg | original_images/love-chicken.jpg
+
+base_translatedimagerendition:
+id  |                file
+-----+-------------------------------------
+183 | images/love-chicken.width-640.jpg
+184 | images/love-chicken.width-720.jpg
+185 | images/love-chicken.width-750.jpg
+186 | images/love-chicken.width-828.jpg
+187 | images/love-chicken.width-1080.jpg
+188 | images/love-chicken.width-1440.jpg
+189 | images/love-chicken.width-2160.jpg
+190 | images/love-chicken.max-165x165.jpg
+```
+
+When displaying images on Janis, you would need to use one of the filepaths that is stored in the `base_translatedimagerendition` table.
+
+#### TODO / Need to Know:
+- graphql does not recognize a foreign relation between `base_translatedimage` and `base_translatedimagerendition`. Where is our graphql schema set up? We'll need to change that, otherwise I'll have the manually parse `base_translatedimage.title` and guess at the base_translatedimagerendition to use.
+- Why was `filename` added to graphql schema for images (joplin/api/schema.py)? How is it different than `title`? Where does it come from?
```

Comment:
"title" can be changed within the Joplin UI, filename doesn't seem to work that way. This is all related to default wagtail image handling.